ci/container_index/engine.py

```diff
@@ -59,23 +60,27 @@ def __init__(
         # Collect the validators that need to run.
         self.validators = []
         # - Schema Validators:
-        if (not (schema_validators and isinstance(schema_validators, list)) or
-                len(schema_validators) <= 0):
-            v_list = config.schema_validators
-        else:
-            v_list = schema_validators
-        self._load_validators(schema_validation, v_list)
+        if schema_validators != "None":
```

Comment:
What would be the scenario for `schema_validators="None"` ?

Reply:
as in, its value would be string `"None"` and not `None` ?

Reply:
If you don't want any of the schema validations to happen for example. Maybe i should make it as some other value like skip? and use None to say use default?

Reply:
If there is no option specified at CLI, it should use the default. Why would a user want to `skip` it?